File: 9.php
<?php
$sentence = "Good Morning Teacher";
$words = explode(" ", $sentence);  

$reverted_sentence = "";

foreach ($words as $word) {
    $reversed_word = "";
    $word_length = strlen($word);

    for ($i = $word_length - 1; $i >= 0; $i--) {
        $reversed_word .= $word[$i];
    }

    $reverted_sentence .= $reversed_word . " ";
}

$reverted_sentence = rtrim($reverted_sentence); 

echo "ประโยคที่ revert : " . $reverted_sentence;
?>

File: 9.cs
using System;

class Program
{
    static void Main()
    {
        string sentence = "Good Morning Teacher";
        string[] words = sentence.Split(' ');

        string revertedSentence = "";

        foreach (string word in words)
        {
            string reversedWord = "";
            int wordLength = word.Length;

            for (int i = wordLength - 1; i >= 0; i--)
            {
                reversedWord += word[i];
            }

            revertedSentence += reversedWord + " ";
        }

        revertedSentence = revertedSentence.TrimEnd();

        Console.WriteLine("ประโยคที่ revert : " + revertedSentence);
    }
}
